dolphinscheduler-task-plugin/dolphinscheduler-task-sql/pom.xml

```diff
@@ -46,6 +46,274 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- jdbc dependency -->
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
```

Comment:
mysql license is not valid for apache-2.0

Reply:
done.